api.swagger.yaml

```diff
@@ -2371,6 +2371,10 @@ components:
           description: This is the standard button border colour. Any CSS colour allowed.
           type: string
           example: '#FFFFFF'
+        standard_button_normal_text_colour
```

Comment:
Shouldn't it be `standard_button_normal_state_text_colour`? Or perhaps just `standard_button_colour` if it's not state-specific.

Also missing a colon.

Reply:
updated